dipy/reconst/tests/test_ivim.py

```diff
@@ -202,6 +202,20 @@ def test_with_higher_S0():
     assert_array_almost_equal(ivim_fit.model_params, params2)
 
 
+def test_b0_threshold_greater_than0():
+    """
+    Added test case for default b0_threshold set to 50.
+    Checks if error is thrown correctly.
+    """
+    bvals_b0t = np.array([50., 10., 20., 30., 40., 60., 80., 100.,
+                          120., 140., 160., 180., 200., 300., 400.,
+                          500., 600., 700., 800., 900., 1000.])
+    N = len(bvals_b0t)
+    bvecs = generate_bvecs(N)
+    gtab = gradient_table(bvals_b0t, bvecs.T)
+    assert_raises(ValueError, IvimModel, gtab)
```

Comment:
Are you sure this is throwing the new error? I suspect it might be throwing this one first: https://github.com/nipy/dipy/blob/master/dipy/reconst/ivim.py#L216

Reply:
@arokem I think you are right! I did not realize this.. Will refactor and update ASAP :)

Reply:
@arokem Checked the error and also added an `assert` to make sure that its the correct error! Does this work?

Reply:
Good stuff! +1 for the merge from me, if the CI comes back green.